package.json

```diff
@@ -181,7 +182,7 @@
 		"publish:dev": "npm run build:packages && lerna publish --npm-tag next",
 		"publish:prod": "npm run build:packages && lerna publish",
 		"test": "concurrently \"npm run lint && npm run test-unit\" \"npm run test-mobile\"",
-		"pretest-e2e": "concurrently \"./bin/reset-e2e-tests.sh\" \"npm run build\"",
+		"pretest-e2e": "concurrently \"./bin/reset-e2e-tests.sh\"",
```

Comment:
I need to revert it, it's painful to build for every run when hacking :)

Reply:
>Maybe we need an option to disable it! 😆

It would be a nice addition. I've worked around this myself by copying, editing, and re-running the subcommand that triggers after the build, but it's not pretty 😅 

e.g.

```
JEST_PUPPETEER_CONFIG=test/e2e/puppeteer.config.js npx wp-scripts test-unit-js --config test/e2e/jest.config.json --runInBand "test/e2e/specs/navigable-toolbar.test.js"                               
```